samples/layouts/tile-manager/layout/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Tile Manager Layout</title>
+  <meta charset="UTF-8" />
+
+  <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+  <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" type="text/css" />
+</head>
+
+<body>
+  <div id="root">
+    <div class="container sample center">
+      <div class="btnWrapper">
+      <igc-button id="saveL">Save Layout</igc-button>
+      <igc-button id="loadL">Load Layout</igc-button>
+      <igc-button id="addT">Add Tile</igc-button>
+      <igc-button id="remT">Remove Tile</igc-button>
+      </div>
+      <igc-tile-manager id="tile-manager1" drag-mode="tile" column-count="2" gap="20px">

```

Comment:
Enabling resizing is probably in order here so they can see spans/sizes are also restored.